region/crateria/central.json

```diff
@@ -5731,18 +5944,16 @@
                     }},
                     {"or": [
                       "canUseFrozenEnemies",
+                      "canTrickyJump",
                       {"and": [
                         "ScrewAttack",
-                        "SpeedBooster"
+                        "SpaceJump"
                       ]},
                       {"and": [
+                        "canCarefulJump",
```

Comment:
I've been assuming `canCarefulJump` is free for g-mode strats, so you could consider doing that and making it a lot simpler. Doesn't matter much either way